delivery_correos_express/models/delivery_carrier.py

```diff
@@ -0,0 +1,260 @@
+            "codDest": "",
+            "nomDest": partner.name or "",  # mandatory
+            "nifDest": "",
+            "dirDest": "".join(streets)[:300],  # mandatory
+            "pobDest": partner.city[:40] or "",  # mandatory
+            "codPosNacDest": partner.zip,  # mandatory
+            "paisISODest": "",
+            "codPosIntDest": "",
+            "contacDest": partner.name[:40] or "",  # mandatory
+            "telefDest": phone[:15],  # mandatory
+            "emailDest": partner.email[:75] or "",
+        }
+
+    def _get_correos_express_sender_info(self, picking):
+        partner = picking.picking_type_id.warehouse_id.partner_id
+        streets = self._get_partner_streets(partner)
+        return {
+            "codRte": self.correos_express_customer_code,
+            "nomRte": partner.name or "",
+            "nifRte": "",
+            "dirRte": "".join(streets)[:300],  # mandatory
+            "pobRte": partner.city,  # mandatory
+            "codPosNacRte": partner.zip,  # mandatory
+            "paisISORte": "",
+            "codPosIntRte": "",
+            "contacRte": partner.name or "",
+            "telefRte": partner.phone or "",
+            "emailRte": partner.email or "",
+        }
+
+    def _get_package_info(self, picking):
+        number_of_packages = picking.number_of_packages or 1
+        package_list = []
+        for i in range(0, number_of_packages):
+            package_list.append(
+                {
+                    "ancho": "",
+                    "observaciones": "",
+                    "kilos": "",
+                    "codBultoCli": "",
+                    "codUnico": "",
+                    "descripcion": "",
+                    "alto": "",
+                    "orden": i + 1,
+                    "referencia": "",
+                    "volumen": "",
+                    "largo": "",
+                }
+            )
+        return number_of_packages, package_list
+
+    def _prepare_correos_express_shipping(self, picking):
+        self.ensure_one()
+        number_of_packages, package_list = self._get_package_info(picking)
+        return dict(
+            {
+                "solicitante": self.correos_express_sender_code,  # mandatory
+                "canalEntrada": "",
+                "numEnvio": "",
+                "ref": picking.name,
+                "refCliente": picking.name,
+                "fecha": fields.Datetime.now().strftime("%d%m%Y"),  # mandatory
+                "contacOtrs": "",
+                "telefOtrs": "",
+                "emailOtrs": "",
+                "observac": "",
+                "numBultos": number_of_packages or 1,  # mandatory
+                "kilos": "%.3f" % (picking.shipping_weight or 1),  # mandatory
+                "volumen": "",
+                "alto": "",
+                "largo": "",
+                "ancho": "",
+                "producto": "93",  # ePaq24  # mandatory
+                "portes": "P",  # mandatory
```

Comment:
```suggestion
                "producto": "93",  # ePaq24  # mandatory
                "portes": "P",  # mandatory
```
Perhaps this could be configured in new fields added in the carrier, right?